MODULE.md

```diff
@@ -0,0 +1,262 @@
+        - [Update Parameters (update-params)](#update-parameters-update-params)
+        - [Stakeholder Payout (stakeholder-payout)](#stakeholder-payout-stakeholder-payout)
+  - [Proof of Authority Module](#proof-of-authority-module)
+    - [Module Functionality](#module-functionality-1)
+      - [Validator Management](#validator-management)
+      - [Staking Parameters Update](#staking-parameters-update)
+      - [Administrative Rights](#administrative-rights)
+      - [Commands](#commands-1)
+        - [Update Parameters (update-params)](#update-parameters-update-params-1)
+        - [Update Staking Parameters (update-staking-params)](#update-staking-parameters-update-staking-params)
+        - [Set Voting Power (set-power)](#set-voting-power-set-power)
+        - [Remove Pending Validator (remove-pending)](#remove-pending-validator-remove-pending)
+        - [Remove Validator (remove)](#remove-validator-remove)
+  - [Token Factory Module](#token-factory-module)
+    - [Module Functionality](#module-functionality-2)
+      - [Token Minting](#token-minting)
+      - [Token Burning](#token-burning)
+      - [Token Administration](#token-administration)
+      - [Metadata Management](#metadata-management)
+      - [Commands](#commands-2)
+        - [Burn (burn)](#burn-burn)
+        - [Burn From (burn-from)](#burn-from-burn-from)
+        - [Mint (mint)](#mint-mint)
+        - [Change Admin (change-admin)](#change-admin-change-admin)
+        - [Create Denom (create-denom)](#create-denom-create-denom)
+        - [Force Transfer (force-transfer)](#force-transfer-force-transfer)
+        - [Modify Metadata (modify-metadata)](#modify-metadata-modify-metadata)
+
+## Manifest Module
+
+The Manifest module is responsible for handling inflation and manual minting events. Below is a structured breakdown of its components and functionalities:
+
+### Module Functionality
+
+Stakeholder Management: Allows the PoA admin to designate stakeholders, who can be one or multiple manifest wallet addresses. These stakeholders are eligible to receive assets issued by the PoA admin.
+
+#### Asset Issuance:
+
+- Manual Issuance: The PoA admin can manually mint and disburse a specified amount of tokens to the stakeholders.
+
+- Automatic Inflation: When enabled, tokens are minted every block, aiming for a predetermined total over a year.
+
+#### Commands
+
+##### Update Parameters (update-params):
+
+- Syntax: `manifestd tx manifest update-params [address:percent_share] [inflation_on_off] [annual_total_mint]`
+
+  - Parameters:
+    - `address:percent_share`: Specifies the wallet address and its share of the total rewards (to the ninth exponent).
+    - `inflation_on_off`: A boolean value (true or false) to toggle automatic inflation.
+    - `annual_total_mint`: The total amount of tokens to be minted annually.
+
+  **Example:** `manifestd tx manifest update-params manifest1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct:100_000_000 false 500000000umfx`
```

Comment:
In this example, you are setting automatic inflation to OFF but still setting the annual total mint. Why?

Add more examples covering more use cases and explain those.

How does one remove a stakeholder? Set share to 0?

E.g.,
- 5 stakeholders (10%, 5%, 40%, 5%, 40%), inflation OFF
- Remove 1 stakeholder
- ...

Reply:
the amount of tokens here could be set to 0umfx. This is due to the CLI requiring a number of arguments for this. In reality, for the multisig a UI or plain JSON would be used. Not the CLI. This CLI is more so for power users, in which we need to give all options for